package.json

```diff
@@ -51,5 +51,9 @@
     "*.{json,md}": [
       "prettier --write"
     ]
+  },
+  "dependencies": {
+    "react": "^18.2.0",
+    "react-dom": "^18.2.0"
```

Comment:
These are new deps, do we need them here?

Reply:
Nope! I tried removing them earlier but I think i had another dodgy dependency that resulted in errors so they ended up staying. Gone for good now!